src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -16,13 +16,16 @@ import * as StringUtility from "Shared/StringUtility";
 import { userContext } from "UserContext";
 import { logConsoleInfo } from "Utils/NotificationConsoleUtils";
 import * as PriorityBasedExecutionUtils from "Utils/PriorityBasedExecutionUtils";
+import { useQueryCopilot } from "hooks/useQueryCopilot";
 import { useSidePanel } from "hooks/useSidePanel";
-import React, { FunctionComponent, useState } from "react";
+import React, { useState } from "react";
+import Explorer from "../../Explorer";
 import { RightPaneForm, RightPaneFormProps } from "../RightPaneForm/RightPaneForm";
 
-export const SettingsPane: FunctionComponent = () => {
+export const SettingsPane = ({ explorer }: { explorer: Explorer }): JSX.Element => {
```

Comment:
We can still keep declaring this as a react FC:
```suggestion
export const SettingsPane: FunctionComponent<{ explorer: Explorer }> = ({ explorer }: { explorer: Explorer }): JSX.Element => {
```